src/__tests__/funding.test.ts

```diff
@@ -84,15 +118,30 @@ export const testFunding = (
     'uplink can stream money to itself'
   )
 
-  await t.notThrowsAsync(
-    withdraw({ uplink, authorize: () => Promise.resolve() }),
-    'withdraws from channel without throwing an error'
-  )
+  const withdrawAmount = uplink.balance$.value
+  const withdrawValueAndFee = await withdrawAndCapture()
 
   t.true(
     uplink.balance$.value.isZero(),
     'balance$ of uplink goes back to zero following a withdraw'
   )
+  const baseBalanceAfterWithdraw = await getBaseBalance(uplink)
+  t.true(
+    baseBalanceAfterWithdraw
+      .minus(baseBalanceAfterDeposit)
+      .isLessThanOrEqualTo(withdrawAmount),
+    'after withdraw, base balance is increased by up to withdraw amount'
```

Comment:
Typo in the assertion message ("by up to"), maybe this instead?
 
```suggestion
    'after withdraw, base balance is increased no more than the withdraw amount'
```